nana/assistant/inline.py

```diff
@@ -266,82 +271,206 @@ async def inline_query_handler(client, query):
+                description="Anime",
+                input_message_content=InputTextMessageContent(text, parse_mode="markdown"),
+                reply_markup=InlineKeyboardMarkup(keyb)))
+        else:
+            answers.append(InlineQueryResultArticle(
+                title="Fabourites",
+                description="Anime",
+                input_message_content=InputTextMessageContent("**No favourites yet!**", parse_mode="markdown")))
+        await client.answer_inline_query(query.id,
+                                        results=answers,
+                                        cache_time=0
+                                        )
+
+    elif string.split()[0] == "cat":
+        image = f"https://d2ph5fj80uercy.cloudfront.net/0{random.randint(1, 6)}/cat{random.randint(0,4999)}.jpg"
+        buttons = [[InlineKeyboardButton("Source", url="https://thiscatdoesnotexist.com/"), InlineKeyboardButton("Refresh", callback_data='cat_pic')]]
+        answers.append(InlineQueryResultPhoto(
+                caption='Hi I like you too >~<',
+                photo_url=image,
+                parse_mode="markdown",
+                title="Cursed Cat",
+                description="Cursed Cat",
+                reply_markup=InlineKeyboardMarkup(buttons)))
+        await client.answer_inline_query(query.id,
+                                        results=answers,
+                                        cache_time=0
+                                        )
+
+    if string.split()[0] == "lookup":
+        if len(string.split()) == 1:
+            await client.answer_inline_query(query.id,
+                                             results=answers,
+                                             switch_pm_text="Search in SpamProtection Database",
+                                             switch_pm_parameter="help_inline"
+                                             )
+            return
+        get_user = string.split(None, 1)[1]
+        try:
+            user = await app.get_chat(get_user)
+        except PeerIdInvalid:
+            await client.answer_inline_query(query.id,
+                                             results=answers,
+                                             switch_pm_text="Can't Find the Chat in Database",
+                                             switch_pm_parameter="help_inline"
+                                             )
+            return
+        api_url = f'https://api.intellivoid.net/spamprotection/v1/lookup?query={user.id}'
+        a = await AioHttp().get_json(api_url)
+        response = a['success']
+        if response == True:
+            date = a["results"]["last_updated"]
+            stats = f'**◢ Intellivoid• SpamProtection Info**:\n'
+            stats += f' - **Updated on**: `{datetime.fromtimestamp(date).strftime("%Y-%m-%d %I:%M:%S %p")}`\n'
+            if a["results"]["attributes"]["is_potential_spammer"] == True:
+                stats += f' - **User**: `USERxSPAM`\n'
```

Comment:
Codacy found an issue: [f-string is missing placeholders (F541)](https://app.codacy.com/gh/MadeByThePinsHub/Nana-Remix/pullRequest?prid=6330175)